denizen_scripts/hub1/repo-link/global_data_handler.dsc

```diff
@@ -122,14 +122,14 @@ Error_Handler:
     - debug record start
   script:
     - ~debug record submit save:mylog
-    - foreach <list[Name|UUID|Server]> as:Tag:
-      - if !<[<[Tag]>].exists>:
+    - foreach Name|UUID|Server as:Tag:
+      - if <[Tag]||false>:
         - foreach next
       - else:
-        - if <[<[Tag]>]> != null:
+        - if <[Tag]> != null:
           - foreach next
       - define WeirdList:->:<[Tag]>
-    - if !<[WeirdList].exists>:
+    - if <[WeirdList].is_empty>:
```

Comment:
`WeirdList` is never pre-defined before the initial foreach loop, we should pre-define `WeirdList` with `<list>` before the foreach in the case the definition never inserts a list

we should add `-define WeirdList <list>` above line 125